.gitignore

```diff
@@ -0,0 +1 @@
+**/*~
```

Comment:
Hi @SoloSynth1, what's the implication of the **/*~ in the .gitignore?

Reply:
Orix:
** will recursively match everything
so basically I am matching every file in every directory that ends with ~
I need that because sometime I use vim for editing the files and it leaves a swap file with a trailing ~ which I don’t want to commit